File: src/main/java/Person.java
import com.fasterxml.jackson.annotation.JsonSubTypes;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;

public abstract class  Person {
    private String fName;
    private String lName;


    public void setfName(String fName){ this.fName = fName;}

    public void setlName(String lName){ this.lName = lName;}

    String getfName() { return fName; }

    String getlName() { return lName; }
}

File: person.py
from abc import ABC


class Person(ABC):
    def __init__(self, f_name, l_name):
        self.f_name = f_name
        self.l_name = l_name

    def get_f_name(self):
        return self.f_name

    def get_l_name(self):
        return self.l_name